### ISSUE TITLE
nginx 404 on login page when running container image

### ISSUE THREAD

**Issue by SalmanAwan** [spotify/backstage#381]
When running inside docker container, cannot navigate to login page due to nginx 404 page error.

## Expected Behavior
Should be able to navigate through internal pages when running inside docker

## Current Behavior
Click on login, shows nginx 404 error page and cannot navigate further

## Possible Solution
Insert nginx site config with redirect rule

## Steps to Reproduce (for bugs)
Step 1: run docker locally
`$ docker build . -t spotify/backstage`
`$ docker run --rm -it -p 80:80 spotify/backstage`

Step 2: Then open http://localhost/ on your browser.
Step 3: Click on login, shows nginx 404 error page and cannot navigate further

## Context
Just trialing out as it looks promising for our team but got blocked on this issue.

## Your Environment
Tried on both Mac OS and RHEL Linux, same behavior.

**Comment by Rugvip** [comment]
Closing this as it's a duplicate of #330 

Currently being fixed in #360
